C_CPP/Data Structures/User Defined Data Structures/Trie/Trie.md

```diff
@@ -0,0 +1,40 @@
+<h1 align="center">  Trie or Prefix Tree </h1>
```

Comment:
avoid using html tags for markdown formatting refer [this](https://www.markdownguide.org/basic-syntax/) instead